Makefile

```diff
@@ -4,12 +4,14 @@ MAKE := make
 RM := rm
 INSTALL := install
 GIT_VERSION := $(shell git describe --abbrev=6 --dirty --always --tags)
+SRC := $(wildcard src/*.c)
+CFLAGS := -O3 -W -Wall $(SRC) -Wwrite-strings -I. -I./include
 
 all: beagle-tester
 
-beagle-tester: beagle-tester.c
-	#$(CC) $(CFLAGS_FOR_BUILD) -W -Wall -Wwrite-strings -O3 -o beagle-tester beagle-tester.c -lroboticscape
-	$(CC) -DVERSION=\"${GIT_VERSION}\" $(CFLAGS_FOR_BUILD) -W -Wall -Wwrite-strings -O3 -o beagle-tester beagle-tester.c
```

Comment:
CFLAGS_FOR_BUILD is a buildroot thing. It might need to come back.

Reply:
Ohh okay, I didn't know that.